root/static/scripts/release-editor/fields.js

```diff
@@ -424,6 +424,10 @@ class Medium {
             return !self.canHaveDiscID() && (self.hasExistingTocs() || hasPregap() || hasDataTracks());
         });
 
+        this.hasTooEarlyFormat = ko.computed(function () {
+            return !!self.formatID() && !!self.release.earliestYear() && (self.release.earliestYear() < MB.formatDates[self.formatID()]);
```

Comment:
Assigning `MB.formatDates[self.formatID()]` to something and checking if it's undefined before doing the comparison would be ideal - although `any_number < undefined` will be false, Flow won't like that sort of thing once we add types.